task/06-conditions-n-loops-tasks.js

```diff
@@ -251,7 +282,19 @@ function reverseInteger(num) {
  *   4916123456789012 => false
  */
 function isCreditCardNumber(ccn) {
-    throw new Error('Not implemented');
+    let sum = 0;
+    ccn = String(ccn);
+    for (let i = 0; i < ccn.length; i++) {
+        let cardNum = parseInt(ccn[i]);
+        if ((ccn.length - i) % 2 === 0) {
+            cardNum *= 2;
+            if (cardNum > 9) {
+                cardNum -= 9;
+            }
+        }
+        sum += cardNum;
+    }
+    return sum % 10 == 0;
```

Comment:
"operator === is always preferable
sometimes ''==' may cause an error
(check at console :
"0" == 0
and "0" === 0)"

Reply:
==  (by value) replaced by === (by type and value)
I see and understand the difference